code/modules/research/designs/supplypacks.dm

```diff
@@ -0,0 +1,2033 @@
+//SUPPLY PACKS
```

Comment:
you seem to have accidentally duped this file

Reply:
I'm honestly not even sure how I managed to pull this off or how to fix this.